client/lib/cart-values/index.js

```diff
@@ -250,9 +250,12 @@ function fillInAllCartItemAttributes( cart, products ) {
 	return update( cart, {
 		products: {
 			$apply: function( items ) {
-				return items.map( function( cartItem ) {
-					return fillInSingleCartItemAttributes( cartItem, products );
-				} );
```

Comment:
This is totally fine to ship, but I'd be inclined to use `Lodash.map` so we can do the more concise 
`( items ) => map( items, ( cartItem ) => fillInSingleCartItemAttributes( cartItem, products ) )`
or even go the step further functional with `partialRight`:
`( items ) => map( items, partialRight( fillInSingleCartItemAttributes, products ) )`).